build.rs

```diff
@@ -11,8 +11,14 @@ fn build_lib() {
         .build();
     println!("cargo:rustc-link-search={}/lib", dst.display());
     println!("cargo:rustc-link-lib=static=treelite_runtime_static");
-    println!("cargo:rustc-link-lib=stdc++");
-    println!("cargo:rustc-link-lib=gomp");
+    #[cfg(target_os = "linux")]
```

Comment:
Does windows require a `cargo:rustc-link-lib` as well?

Reply:
toolchain=msvc: Windows requires MSVC runtime. It works fine without a configuration like "cargo:rustc-link-lib=stdc++".

Reply:
That's cool!